pygmt/tests/test_clib_put_matrix.py

```diff
@@ -85,3 +86,23 @@ def test_put_matrix_grid():
                 # Load the data and check that it's correct
                 newdata = tmp_file.loadtxt(dtype=dtype)
                 npt.assert_allclose(newdata, data)
+
+            # Save the data to a netCDF grid and check
```

Comment:
Took me a while to remember what this was for! Just a few docstring suggestions.

```suggestion
            # Save the data to a netCDF grid and check that xarray can load it
```

Also change at line 59:

```
    "Check that assigning a numpy 2d array to an ASCII and NetCDF grid works"
```